### PR TITLE
 Full text search URL is missing in autocomplete  FIX: #196 

### PR DESCRIPTION
**Pull request by adriantintpilver**

### CODE REVIEW

File: site/index.scroll [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import settings.scroll
 
+htmlTitle PLDB - A Programming Languages Database
```
Comment (breck7) [inline site/index.scroll:3]: this should be in a different commit

Comment (breck7) [inline site/index.scroll:3]: but i'm curious, what does this do?

I see the title for the homepage is already `<title>PLDB: A Programming Language Database</title>`, why add this line?

File: site/search.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ const initSearchAutocomplete = elementId => {
         label: `Full text search for "${query.replace(/</g, "&lt;")}"`,
         appeared: "",
         id: "",
-        url: "/search?q="
+        url: `/search?q="${query.replace(/</g, "&lt;")}"`
```
Comment (breck7) [inline site/search.js:32]: oh nice! I think you figured it out.

File: site/index.scroll (hunk 1 above)
Comment (adriantintpilver) [reply site/index.scroll:3]: ho no, is my mistake. pleas ignore this file, only search.js is the important change for this issue. The file index.scroll change is for other issue. sorry.

Comment (breck7) [reply site/index.scroll:3]: ok please update this pull request or create a new PR. thanks!

**Comment by adriantintpilver** [comment]
I cancel this pull request and create a new with search.js file only. --> https://github.com/breck7/pldb/pull/256
